Cargo.toml

```diff
@@ -27,6 +33,9 @@ quickcheck_macros = "0.8"
 itertools = "0.8"
 criterion = "0.2.11"
 
+[dev-dependencies.serde]
+version = "1.0"
```

Comment:
What is this for?

Reply:
seems its not needed, its leftover from when I was figuring something out.